librocksdb_sys/Cargo.toml

```diff
@@ -23,6 +23,7 @@ sse = ["libtitan_sys/sse"]
 [build-dependencies]
 cc = "1.0.3"
 cmake = "0.1"
+bindgen = "0.52"
```

Comment:
seems bindgen 0.52 is conflicting with grpcio, which is using bindgen 0.51. Any thoughts on how to fix?
<img width="789" alt="Screen Shot 2019-12-18 at 4 43 12 PM" src="https://user-images.githubusercontent.com/2606959/71135089-8311cb00-21b5-11ea-8653-760b6985f608.png">

Reply:
use bindgen 0.51 to fix it ?